src/module.ts

```diff
@@ -41,6 +41,7 @@ export default defineNuxtModule<ModuleOptions>({
     colorModePreferenceCookieName: "naive_color_mode_preference",
     iconSize: 20,
     iconDownload: false,
+    iconCollectionsUrl: 'https://raw.githubusercontent.com/iconify/icon-sets/master/collections.json',
```

Comment:
Is this right? I'm not sure

Reply:
No problem, I will set the current one as it solves the issue in some cases.